Library/Homebrew/cmd/help.rb

```diff
@@ -39,4 +39,24 @@ def help
   def help_s
     HOMEBREW_HELP
   end
+
+  def help_cmd(cmd)
+    if File.exist?(HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.sh")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.sh"
+    elsif ARGV.homebrew_developer? && File.exist?(HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.sh")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.sh"
+    elsif File.exist?(HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb"
+    elsif ARGV.homebrew_developer? && File.exist?(HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.rb")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.rb"
+    else
+      return ""
+    end
+
+    cmd_path.read.
+      split("\n").
+      grep(/^#:/).
+      map { |line| line.slice(2..-1).delete("`").sub(/^  \* /, "") }.
```

Comment:
I wonder if `.sub(/^  \* /, "")` should be `.sub(/^  \* /, "brew ")` instead? Now we're getting, e.g.:

```
$ brew help cat
cat <formula>:
    Display the source to <formula>.
```

… and it feels a bit weird that there's no prefixed `brew` (while in the context of the man page it makes sense). One thing I'm not entirely sure about if that will create confusion for commands with aliases, e.g. printing `brew ls, list` instead of just `ls, list`.